### PR TITLE
Fix CPU gauge to just use normalized CPU percentages

### PR DESCRIPTION
**Pull request by fearful-symmetry**
## What does this PR do?

This alters the CPU Usage Gauge to just use normalized values for visualizations, and removes the painless script, which was (incorrectly) dividing the normalized value by the CPU count.

I'd actually like some feedback on this, since I can't figure out why the visualization hasn't *always* been like this. We've been doing some math cleverness to take the "regular" CPU value and divide it by the CPU count, when this is basically what the `norm.cpu` values do to begin with. I've tested this change and it works fine, but I'm a tad paranoid and I wonder if there's a reason why we haven't been doing this before.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.

## How to test this PR locally

Pull down, build. Find the CPU Usage gauge In the System Integration Host Details dashboard. Make sure the gauge is populated and the value is correct.

### CODE REVIEW

File: packages/system/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.1.4"
+  changes:
+    - description: Fix issue with normalized CPU gauge
+      type: enhancement
```
Comment (mtojek) [inline packages/system/changelog.yml:5]: I think this is more like a "bugfix" than "enhancement".

**Comment by exekias** [comment]
> I'd actually like some feedback on this, since I can't figure out why the visualization hasn't _always_ been like this. We've been doing some math cleverness to take the "regular" CPU value and divide it by the CPU count, when this is basically what the `norm.cpu` values do to begin with. I've tested this change and it works fine, but I'm a tad paranoid and I wonder if there's a reason why we haven't been doing this before.

@jsoriano do you happen to remember this? I don't 🤔 , I guess this is because normalized values were added after the initial dashboards were created

**Review by exekias — APPROVED** [review]
Change LGTM, besides comment on change type in the changelog

**Comment by andrewkroh** [comment]
> I guess this is because normalized values were added after the initial dashboards were created

Or maybe that `normalized_percentages` are disabled by default. But here we have control of the [`cpu.metrics` setting](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-system-cpu.html#_configuration_4).

**Comment by fearful-symmetry** [comment]
> Or maybe that normalized_percentages are disabled by default. But here we have control of the cpu.metrics setting.

So, @andrewkroh , everything is enabled by default save for `ticks`, which makes it even weirder.

> I guess this is because normalized values were added after the initial dashboards were created
Honestly, that's my best guess.

**Comment by andrewkroh** [comment]
> everything is enabled by default save for ticks

I think the docs are wrong then. They say:

> The default value is cpu.metrics: [percentages].

**Comment by fearful-symmetry** [comment]
> The default value is cpu.metrics: [percentages].

@andrewkroh where are you seeing that? I'm grepping through the repo and can't find it.

**Comment by andrewkroh** [comment]
That's from the Metricbeat docs for `cpu.metrics`. https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-system-cpu.html#_configuration_4

**Comment by jsoriano** [comment]
> > I guess this is because normalized values were added after the initial dashboards were created
> 
> Or maybe that `normalized_percentages` are disabled by default. But here we have control of the [`cpu.metrics` setting](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-system-cpu.html#_configuration_4).

Yes, I think the reason is this one, that normalized values are (or were) disabled by default.

**Comment by fearful-symmetry** [comment]
Yah, key word being _were_ here. We might want to look into just refreshing most of the system dashboards. A lot of this stuff looks really old.

**Pull request merged**